CRITICAL - 2024-12-10 15:28:04 --> Error: Class "App\Controllers\KontakPesanModel" not found
[Method: POST, Route: contact/save]
in APPPATH\Controllers\Home.php on line 28.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->saveMessage()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-12-10 15:34:36 --> mysqli_sql_exception: Unknown column 'no_telp' in 'field list' in C:\xampp-new\htdocs\freelancer\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp-new\htdocs\freelancer\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `da...', 0)
#1 C:\xampp-new\htdocs\freelancer\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `da...')
#2 C:\xampp-new\htdocs\freelancer\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `da...')
#3 C:\xampp-new\htdocs\freelancer\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `da...', Array, false)
#4 C:\xampp-new\htdocs\freelancer\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp-new\htdocs\freelancer\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp-new\htdocs\freelancer\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp-new\htdocs\freelancer\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp-new\htdocs\freelancer\app\Controllers\Home.php(44): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp-new\htdocs\freelancer\system\CodeIgniter.php(933): App\Controllers\Home->saveMessage()
#10 C:\xampp-new\htdocs\freelancer\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#11 C:\xampp-new\htdocs\freelancer\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp-new\htdocs\freelancer\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp-new\htdocs\freelancer\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp-new\htdocs\freelancer\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-12-10 15:34:36 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'no_telp' in 'field list'
[Method: POST, Route: contact/save]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (:nama:, :email:, :no_telp:, :pesan:, :delete_data:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Home.php(44): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->saveMessage()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-12-10 15:34:36 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'no_telp' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (\'indra\', \'indratesbkn_v0.0@yopmail.com\', \'082131361547\', \'apa saya bisa hubungi\', 0)')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (\'indra\', \'indratesbkn_v0.0@yopmail.com\', \'082131361547\', \'apa saya bisa hubungi\', 0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (:nama:, :email:, :no_telp:, :pesan:, :delete_data:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Home.php(44): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->saveMessage()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-12-10 15:34:36 --> [Caused by] mysqli_sql_exception: Unknown column 'no_telp' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (\'indra\', \'indratesbkn_v0.0@yopmail.com\', \'082131361547\', \'apa saya bisa hubungi\', 0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (\'indra\', \'indratesbkn_v0.0@yopmail.com\', \'082131361547\', \'apa saya bisa hubungi\', 0)')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (\'indra\', \'indratesbkn_v0.0@yopmail.com\', \'082131361547\', \'apa saya bisa hubungi\', 0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (:nama:, :email:, :no_telp:, :pesan:, :delete_data:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Home.php(44): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->saveMessage()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-12-10 15:34:48 --> mysqli_sql_exception: Unknown column 'no_telp' in 'field list' in C:\xampp-new\htdocs\freelancer\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp-new\htdocs\freelancer\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `da...', 0)
#1 C:\xampp-new\htdocs\freelancer\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `da...')
#2 C:\xampp-new\htdocs\freelancer\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `da...')
#3 C:\xampp-new\htdocs\freelancer\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `da...', Array, false)
#4 C:\xampp-new\htdocs\freelancer\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp-new\htdocs\freelancer\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp-new\htdocs\freelancer\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp-new\htdocs\freelancer\system\BaseModel.php(751): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp-new\htdocs\freelancer\app\Controllers\Home.php(44): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp-new\htdocs\freelancer\system\CodeIgniter.php(933): App\Controllers\Home->saveMessage()
#10 C:\xampp-new\htdocs\freelancer\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#11 C:\xampp-new\htdocs\freelancer\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp-new\htdocs\freelancer\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#13 C:\xampp-new\htdocs\freelancer\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp-new\htdocs\freelancer\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}
CRITICAL - 2024-12-10 15:34:48 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'no_telp' in 'field list'
[Method: POST, Route: contact/save]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (:nama:, :email:, :no_telp:, :pesan:, :delete_data:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Home.php(44): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->saveMessage()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-12-10 15:34:48 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'no_telp' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (\'indra\', \'indratesbkn_v0.0@yopmail.com\', \'082131361547\', \'aa\', 0)')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (\'indra\', \'indratesbkn_v0.0@yopmail.com\', \'082131361547\', \'aa\', 0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (:nama:, :email:, :no_telp:, :pesan:, :delete_data:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Home.php(44): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->saveMessage()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-12-10 15:34:48 --> [Caused by] mysqli_sql_exception: Unknown column 'no_telp' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (\'indra\', \'indratesbkn_v0.0@yopmail.com\', \'082131361547\', \'aa\', 0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (\'indra\', \'indratesbkn_v0.0@yopmail.com\', \'082131361547\', \'aa\', 0)')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (\'indra\', \'indratesbkn_v0.0@yopmail.com\', \'082131361547\', \'aa\', 0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_kontak_pesan` (`nama`, `email`, `no_telp`, `pesan`, `delete_data`) VALUES (:nama:, :email:, :no_telp:, :pesan:, :delete_data:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(751): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Home.php(44): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Home->saveMessage()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
